ProcessMaker/processmaker
# FOUR-13098 | User With Permissions to Create Processes Does Not Have the Processes Option in Sidebar Menu

# Issue
Ticket: [FOUR-13098](https://processmaker.atlassian.net/browse/FOUR-13098)

When an Admin user creates a user with Create, View, or Edit Process permissions, and logs in with the user from the same browser window, the Processes option does not appear in the Designer sidebar menu.

# Solution
Clear out the session permissions whenver a user logs out.

# How to Test
1. Go to branch `observation/FOUR-13098` in `processmaker`.
2. Log in with an Admin user.
3. Create a User that has Create, Edit, and View Process permissions.
4. Logout of the Admin User account. From the same tab, login with the created User account.
5. Go to Designer.
6. The Processes option should display in the sidebar menu.

ci:next

# Code Review Checklist


[FOUR-13098]: https://processmaker.atlassian.net/browse/FOUR-13098?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
